app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -262,16 +282,16 @@ function ContainerTopologyCtrl($scope, $http, $interval, $location, topologyServ
   };
 
   $scope.searchNode = function() {
-    var svg = topologyService.getSVG(d3);
-    var query = $scope.search.query;
+    var svg = topologyService.getSVG($scope.d3);
+    var query = $(search)[0].value;
 
    topologyService.searchNode(svg, query);
   };
 
   $scope.resetSearch = function() {
-    topologyService.resetSearch(d3);
+    topologyService.resetSearch($scope.d3);
 
     // Reset the search term in search input
-    $scope.search.query = "";
+    $(search)[0].value = "";
   };
 }
```

Comment:
I cheated here and used some jQuery to get the toolbar to function as expected @himdel @martinpovolny